dynamiqs/integrators/apis/dssesolve.py

```diff
@@ -0,0 +1,211 @@
+from __future__ import annotations
+
+from jaxtyping import ArrayLike, PRNGKeyArray
+
+from ...gradient import Gradient
+from ...options import Options
+from ...qarrays.qarray import QArrayLike
+from ...result import DSSESolveResult
+from ...solver import Solver
+from ...time_qarray import TimeQArray
+
+
+def dssesolve(
+    H: QArrayLike | TimeQArray,
+    jump_ops: list[QArrayLike | TimeQArray],
+    psi0: QArrayLike,
+    tsave: ArrayLike,
+    keys: PRNGKeyArray,
+    *,
+    exp_ops: list[QArrayLike] | None = None,
+    solver: Solver | None = None,
+    gradient: Gradient | None = None,
+    options: Options = Options(),  # noqa: B008
+) -> DSSESolveResult:
+    r"""Solve the diffusive stochastic Schrödinger equation (SSE).
+
+    Warning:
+        This function has not been implemented yet. The following API is indicative
+        of the planned implementation.
+
+    In quantum optics, the diffusive SSE describes the evolution of a quantum
+    system under homodyne or heterodyne detection with unit detection efficiency.
+    This function computes the evolution of the state vector $\ket{\psi(t)}$ at time $t$,
+    starting from an initial state $\ket{\psi_0}$ according to the diffusive SSE in Itô
+    form ($\hbar=1$, time is implicit(1))
+    $$
+        \begin{split}
+            \dd\\!\ket\psi = \Bigg[
```

Comment:
We should either do the `\\!` everywhere in the doc or nowhere. If we choose to do it I suggest doing it in another PR.

Reply:
What do you mean? There was some very wierd spacing issue in this equation, which is not the case in other equations.

Reply:
Ha I don't have the issues on my end. 🤔 I thought you just wanted to have the straight "d" closer to the ket symbol.

Reply:
Yes that's exactly what I wanted to do. The "d" looked very far from the ket symbol if I didn't add the `\\!`, for some reason?

Reply:
Ok, I'm not sure it's needed anymore then.